widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -9325,3 +9393,64 @@ body.bootstrap .modal.confirmationDialog .dialog div.body {
     text-overflow: ellipsis;
   }
 }
+
+/* endregion Main Toolbar */
+
+.schedulesPanel .toolbar {
+  background: none;
+  border: 0;
+  margin-bottom: 5px;
+}
+
+.schedulesPanel .pentaho-table th {
+  font-size: 12px;
+  border-spacing: 0;
+  color: #FFF;
+  padding: 8px 10px 10px 10px;
+  font-weight: normal;
+}
+
+#generated-content-location {
+  flex: auto;
+  margin-bottom: unset;
+  border-radius: 4px;
+}
+
+.schedule-label {
+  margin-top: unset;
+  margin-bottom: unset;
+  color: #666666;
+}
+
+/* NOTE: --flex-items is not working for contained Flex Panels */
+.schedule-output-location-dialog .dialog-content > table > tbody > tr > td > * {
+  flex: none;
+}
+
+#schedule-name-input {
+  width: unset;
+  margin-bottom: unset;
+  flex: 4 1 0;
+}
+
+.schedule-timestamp-listbox {
+  margin-left: unset;
+  margin-bottom: unset;
+  min-width: 200px;
+  width: unset;
+}
+
+.schedule-name-panel {
+  flex-wrap: wrap;
+  --flex-items: 1 1 0;
+}
+
+.schedule-caption-panel {
+  margin-top: unset;
+}
+
+.schedule-dialog-button-panel {
+  padding-right: 3px;
+  padding-bottom: 2px;
+  width: 100%;
+}
```

Comment:
Should be moved to the above group `/* region Schedule Wizard Dialogs */`, and each rule placed under the appropriate sub-section, if any.